tests/apps_test.go

```diff
@@ -145,9 +145,7 @@ var _ = Describe("Apps", func() {
 			sess, err := start("deis info -a %s", appName)
 			Expect(err).NotTo(HaveOccurred())
 			Eventually(sess).Should(Say("=== %s Processes", appName))
-			Eventually(sess).Should(SatisfyAny(
-				Say("web.1 initialized"),
-				Say("web.1 up")))
+			Eventually(sess).Should(Say(`%s-v\d-[\w-]+ up \(v\d\)`, appName))
```

Comment:
Yes--I'll do some refactoring on #53 to reduce the code duplication and move the regex to package level.